web/src/app/playController.js

```diff
@@ -403,21 +392,23 @@ angular
                     movable: {
                         color: chessToColor(chess),
                         dests: chessToDests(chess)
-                    }
+                    },
+                    check: false
```

Comment:
Right, because this function later does its own `in_check()` test, here `chessground` can be provided `check: false`.  That could explain a long-standing double-check-sound bug...

Reply:
Yes. Prior to each move we clear the previous check state and then test for it ourselves. Also, when I moved to Webpack, the old version of Chessground lying in the repo was removed and we now use the latest. In the newest version `setCheck()` is no longer exposed in the public API, so it needed to be replaced. I did it for `playAIController` initially, but forgot about `playController`.